### DISCUSSION TITLE
Como atualizar um repositório do Github com o master?

### DISCUSSION THREAD

**Discussion by Marcelosilva10** [frontendbr/forum#148] [Perguntas e Respostas]
~Pessoas bom dia a todos primeiramente sou novato ainda com o Github queria saber como faço para atualizar o meu fork no Github, se alguém puder me ajudar?

**Comment by danilovaz** [comment]
o @fdaciuk fez um post muito bom sobre isso, dá uma olhada [aqui](http://blog.da2k.com.br/2014/01/19/manter-repositorio-github-forkado-sincronizado-com-o-original/)

**Comment by Marcelosilva10** [comment]
Vlw @danilovaz viu eu apaguei o meu repositório antigo e forkar denovo assim da também?

**Comment by danilovaz** [comment]
@Marcelosilva10 dá sim. só que eu acho que é um puta trampo. o melhor mesmo é você atualizar o fork igual é explicado no post, dessa forma você consegue pegar repositórios antigos na sua máquina e atualizar com o repo original, acho mais fácil assim

**Comment by evansantos** [comment]
@Marcelosilva10 é mais esquema fazer do jeito que está no post do @fdaciuk, deletar e copiar é muito cornojob dependendo do projeto você tem que ficar instalando dependência não é legal! 

Só não esquece de sempre dar `pull upstream/master` pra atualizar o projeto senão dá ruim quando você commitar 

![cup-golf-pet-YbK5VsLYhCqHe](https://camo.githubusercontent.com/5582a70212d6479b768240537e5f9947386bc281/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f59624b3556734c5968437148652f67697068792e676966)

**Comment by Marcelosilva10** [comment]
Pqp, já fiz o estilo Gabit do ex man apaguei. mais vlw pelo blog

**Comment by felipejesus-front** [comment]
Não conheço vocês, mas já considero pra caramba. <3 

Valeu a ajuda tbm.
